api/src/main/java/org/openmrs/api/db/ConceptDAO.java

```diff
@@ -321,64 +318,87 @@ public Integer getCountOfConcepts(String phrase, List<Locale> locales, boolean i
 	public Iterator<Concept> conceptIterator();
 	
 	/**
-	 * @see org.openmrs.api.ConceptService#getConceptsByMapping(java.lang.String, java.lang.String)
+	 * @see org.openmrs.api.ConceptService#getConceptsByMapping(String, String, boolean)
 	 */
 	public List<Concept> getConceptsByMapping(String code, String sourceName, boolean includeRetired);
 	
 	/**
-	 * @param uuid
-	 * @return concept or null
+	 * @see org.openmrs.api.ConceptService#getConceptByUuid(java.lang.String)
 	 */
 	public Concept getConceptByUuid(String uuid);
 	
 	/**
-	 * @param uuid
-	 * @return concept class or null
+	 * @see org.openmrs.api.ConceptService#getConceptClassByUuid(java.lang.String)
 	 */
 	public ConceptClass getConceptClassByUuid(String uuid);
 	
+	/**
+	 * @see org.openmrs.api.ConceptService#getConceptAnswerByUuid(java.lang.String)
+	 */
 	public ConceptAnswer getConceptAnswerByUuid(String uuid);
 	
+	/**
+	 * @see org.openmrs.api.ConceptService#getConceptNameByUuid(java.lang.String)
+	 */
 	public ConceptName getConceptNameByUuid(String uuid);
 	
+	/**
+	 * @see org.openmrs.api.ConceptService#getConceptSetByUuid(java.lang.String)
+	 */
 	public ConceptSet getConceptSetByUuid(String uuid);
 	
+	/**
+	 * @see org.openmrs.api.ConceptService#getConceptSourceByUuid(java.lang.String)
+	 */
 	public ConceptSource getConceptSourceByUuid(String uuid);
 	
 	/**
-	 * @param uuid
-	 * @return concept data type or null
+	 * @see org.openmrs.api.ConceptService#getConceptDatatypeByUuid(java.lang.String)
 	 */
 	public ConceptDatatype getConceptDatatypeByUuid(String uuid);
 	
 	/**
-	 * @param uuid
-	 * @return concept numeric or null
+	 * @see org.openmrs.api.ConceptService#getConceptNumericByUuid(java.lang.String)
 	 */
 	public ConceptNumeric getConceptNumericByUuid(String uuid);
 	
 	/**
-	 * @param uuid
-	 * @return concept proposal or null
+	 * @see org.openmrs.api.ConceptService#getConceptProposalByUuid(java.lang.String)
 	 */
 	public ConceptProposal getConceptProposalByUuid(String uuid);
 	
 	/**
-	 * @param uuid
-	 * @return drug or null
+	 * @see org.openmrs.api.ConceptService#getDrugByUuid(java.lang.String)
 	 */
 	public Drug getDrugByUuid(String uuid);
 	
+	/**
+	 * @see org.openmrs.api.ConceptService#getDrugIngredientByUuid(java.lang.String)
+	 */
 	public DrugIngredient getDrugIngredientByUuid(String uuid);
 	
+	/**
+	 * Returns a map of all concepts, wherein the ConceptID is mapped to the uuid of 
```

Comment:
Where did you get these from? I thought you would use the see annotation like you have done above.

Reply:
@dkayiwa This function is not found in ConceptService.java. I have been instructed by @samuelmale to simply document them, based on their function in HibernateConceptDAO.java. :)